### PR TITLE
Clean up space limit constants and increase it to 250 MB

### PR DESCRIPTION
**Pull request by wojtekn**
Related to https://github.com/Automattic/dotcom-forge/issues/8515

## Proposed Changes

I propose increasing the demo size limit to 250 MB and cleaning up constants to avoid calculating values twice.

### Happy path

1. Create a site in Studio
2. Upload 100-150MB of media to the site
3. Add a demo site
4. Confirm that demo site is created successfully

### Error

1. Create a site in Studio
2. Upload >250MB of media to the site
3. Add a demo site
4. Confirm error message shows that limit of 250 MB was exceeded

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

**Review by kozer — APPROVED** [review]
Tested it and works as expected! Nice work Wojtek!
image

File: src/hooks/use-archive-site.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ export function useArchiveSite() {
 						__(
 							'The site exceeds the maximum size of %dMB. Please remove some files and try again.'
 						),
-						Math.floor( LIMIT_ARCHIVE_SIZE / 1024 / 1024 )
+						Math.floor( SIZE_LIMIT_MB )
```
Comment (sejas) [inline src/hooks/use-archive-site.ts]: Should we remove the Math.floor ?
```suggestion
						SIZE_LIMIT_MB
```

**Review by sejas — APPROVED** [review]
Looks good! Thanks for increasing the snapshots size. I confirm the alert appears when the wp-content zip file is bigger than 250MB and it works if its smaller.

I've added a suggestion about removing `Math.floor`.


https://github.com/user-attachments/assets/31e12998-bded-4a15-ac30-c2dee3faa2fc

File: src/hooks/use-archive-site.ts (hunk 1 above)
Comment (wojtekn) [reply src/hooks/use-archive-site.ts]: Done: https://github.com/Automattic/studio/pull/446/commits/daa91834e8fb78f0eeb6048d63306d7953a5e6e2

**Pull request merged**